### PR TITLE
Added cron role

### PR DESCRIPTION
**Pull request by brettminnie**

### CODE REVIEW

File: roles/cron/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+- name: Set cron_paths fact
+  set_fact:
+    cron_paths: "/usr/local/bin:/usr/bin:/bin:/usr/sbin"
+
+- name: add cron locations to path
+  lineinfile:
+    dest: /etc/environment
+    state: present
+    backrefs: yes
+    regexp: "PATH=([\"]*)((?!.*?{{ cron_paths }}).*?)([\"]*)$"
```
Comment (carinadigital) [inline roles/cron/tasks/main.yml]: Why the change of environment?

Comment (brettminnie) [reply roles/cron/tasks/main.yml]: Ensuring our paths are in the environment, though they should be, I'll remove it

File: roles/cron/tasks/cron.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  when: job.enabled
+
+- block:
+    - name: Remove cron job
+      cron:
+        name: "{{ job.name }}"
+        user: "{{ job.user }}"
+        state: absent
+      become: true
+  when: job.enabled and not job.enabled
```
Comment (Azulinho) [inline roles/cron/tasks/cron.yml]: I may need some coffee.

can you explain me that when statement?

**Review by Azulinho — COMMENTED** [review]
confused about your when statement, that's my only question about this role

File: roles/cron/tasks/cron.yml (hunk 2 above)
Comment (brettminnie) [reply roles/cron/tasks/cron.yml]: Needs cleaning up :)
